### PR TITLE
docs(samples): Add samples for Cloud Spanner Default Backup Schedules

### LINKED ISSUE #1237: Add samples for Cloud Spanner Default Backup Schedules
Cloud Spanner Default Backup Schedules adds a new field default_backup_schedule_type to google/spanner/admin/instance/v1/spanner_instance_admin.proto.

Add create and update instance code samples to include setting & updating this field.

### PR DESCRIPTION
**Pull request by Sally-Ye**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [X] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-spanner/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [X] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #1237 🦕

### CODE REVIEW

File: samples/samples/snippets.py [hunk 1]
```diff
@@ -3222,6 +3222,57 @@ def create_instance_with_autoscaling_config(instance_id):
...
+    )
+
+    operation = spanner_client.instance_admin_api.create_instance(
+      parent=spanner_client.project_name,
+      instance_id=instance_id,
+      instance=spanner_instance_admin.Instance(
+          config=config_name,
+          display_name="This is a display name.",
+          node_count=1,
+          edition=spanner_instance_admin.Instance.DefaultBackupScheduleType.NONE,  # Optional
```
Comment (harshachinta) [inline samples/samples/snippets.py:3239]: should this be DefaultBackupScheduleType?

File: samples/samples/snippets.py [hunk 2]
```diff
@@ -3222,6 +3222,57 @@ def create_instance_with_autoscaling_config(instance_id):
...
+# [START spanner_update_instance_default_backup_schedule_type]
+def update_instance_default_backup_schedule_type(instance_id):
+    spanner_client = spanner.Client()
+
+    name = "{}/instances/{}".format(spanner_client.project_name, instance_id)
+
+    operation = spanner_client.instance_admin_api.update_instance(
+      instance=spanner_instance_admin.Instance(
+          name=name,
+          edition=spanner_instance_admin.Instance.DefaultBackupScheduleType.AUTOMATIC,  # Optional
```
Comment (harshachinta) [inline samples/samples/snippets.py:3261]: same here - default_backup_schedule_type

**Comment by harshachinta** [comment]
@Sally-Ye Please fix samples lint.
image

**Comment by Sally-Ye** [comment]
@harshachinta Done.  Thanks!

**Review by harshachinta — APPROVED** [review]

**Pull request merged**